web/core/local-db/worker/wa-sqlite/src/sqlite-api.js

```
@@ -0,0 +1,899 @@
+  })();
+
+  sqlite3.column = function (stmt, iCol) {
+    verifyStatement(stmt);
+    const type = sqlite3.column_type(stmt, iCol);
+    switch (type) {
+      case SQLite.SQLITE_BLOB:
+        return sqlite3.column_blob(stmt, iCol);
+      case SQLite.SQLITE_FLOAT:
+        return sqlite3.column_double(stmt, iCol);
+      case SQLite.SQLITE_INTEGER:
+        const lo32 = sqlite3.column_int(stmt, iCol);
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)